es-vue-base/src/lib-components/EsAccordion.vue

```diff
@@ -1,18 +1,24 @@
 <template>
     <div class="EsAccordion border-bottom border-light rounded-bottom">
-        <header role="tab">
-            <es-button
-                class="EsAccordion-button h-auto align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 rounded-0 text-body text-left text-decoration-none w-100"
+        <header
+            role="tab"
+            class="position-relative d-inline-block w-100">
+            <component
+                :is="headingTag"
+                class="EsAccordion-heading mb-0 h-auto align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 rounded-0 text-body text-left text-decoration-none"
                 :class="{
-                    'bg-light': isVisible,
+                    'bg-gray-200': isVisible,
                     'bg-white': !isVisible,
-                    'EsAccordion-button--visible': isVisible,
-                }"
-                variant="link"
-                @click="handleClick">
+                    'EsAccordion-heading--visible': isVisible,
+                }">
                 <slot name="title" />
                 <icon-chevron-down class="EsAccordion-icon flex-shrink-0 ml-200" />
-            </es-button>
+            </component>
+            <button
+                class="EsAccordion-button position-absolute w-100 h-100 border-0 bg-transparent"
+                @click="handleClick">
+                <span class="d-none">{{ isVisible ? 'collapse' : 'expand' }}</span>
```

Comment:
If we replace the `d-none` here with the `sr-only` class, this text will be readable by screen readers, but still visually hidden, via some `position: absolute` magic. Screen readers don't read out elements that are `display: none`.